### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by AngelOnFira** [facebook/fbctf#688]
Hey, just wanted to check what the future of this project will be. The last commit on master is over a year old, and there are 22 open PRs. I would like to adopt this scoreboard as my main, however, if development is stagnant, then it is not as enticing. Any info would be great!

**Comment by garosenb** [comment]
I was wondering the same thing.

**Comment by punkdis** [comment]
I don't think it is a dead project as what features are people wanting to incorporate into the product.

**Comment by m1ch41** [comment]
@justinwray Can you give as clear answer ? Dead or not
